OneSignalSDK/onesignal/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="location_not_available_title">Location Not Available</string>
+    <string name="location_not_available_message">Looks like this app doesn\'t have location permissions granted</string>
```

Comment:
Granted isn't correct. Permission isn't present what we want. We don't want to show any message if the permission was simply declined.

Reply:
Good catch, this string should be the one that is on the screenshot, that one is the one that Lee defined